src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -169,13 +170,16 @@ export default class UniversalDiscoveryModule extends Component {
         this.props.onConfirm(this.addContentTypeInfo([location]));
     }
 
-    addContentTypeInfo(content) {
-        return content.map((item) => {
-            item = JSON.parse(JSON.stringify(item));
+    addContentTypeInfo(contents) {
```

Comment:
`content` is uncountable. Maybe you should use `contentMeta`

Reply:
I would use generic `items`